example/timer/odp_timer_accuracy.c

```diff
@@ -358,14 +422,41 @@ static int start_timers(test_global_t *test_global)
 		timer_param.max_tmo = test_global->opt.max_tmo_ns;
 	}
 
+	if (mode == MODE_PERIODIC) {
+		odp_timer_periodic_capability_t capa;
+
+		capa.base_freq_hz = freq;
+		capa.max_multiplier = max_multiplier;
+		capa.res_ns = res_ns;
+
+		if (odp_timer_periodic_capability(test_global->opt.clk_src, &capa) < 0) {
```

Comment:
It would be good to handle also return value 0. Maybe print a warning that the requested base frequency is not met, which causes a drift. 

 * @retval 0   Success. Capability does not match base frequency exactly, but meets or exceeds
 *             other requested values.